_posts/2015-01-30-cloudpipe.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Put that in your cloud and smoke it"
```

Comment:
Hahahaha. I was on the fence about it. I'll probably change it as soon as I can think about something better.

Maybe I'll be boring and just use "Cloudpipe".